ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -23,6 +30,22 @@ StatusDialog {
         color: Theme.palette.baseColor3
     }
 
+    header: AccountsModalHeader {
```

Comment:
I'm also using temporary the `AccountsModalHeader` in the dApps workflow. I'm wondering if requirements for swap are similar and we can share the reworked control in the future. Here it is what we expect from it to look like:
![image](https://github.com/status-im/status-desktop/assets/47554641/d298367a-8dde-4aa1-918f-2fbc402f5f10)

Reply:
@stefandunca definitely seems like it would be the same indeed. Link to UI task here https://github.com/status-im/status-desktop/issues/14779